File: scripts/inspect_excel.py
import openpyxl
import pandas as pd
import json

excel_path = "Acompanhamento de baixa - MODENS IHS.xlsx"

try:
    xl = pd.ExcelFile(excel_path)
    print("Sheets in Excel:", xl.sheet_names)
    
    for sheet in xl.sheet_names:
        df = xl.parse(sheet, nrows=5)
        print(f"\n--- Sheet: {sheet} (First 5 rows) ---")
        print(df.to_string())
        print("Columns:", list(df.columns))
except Exception as e:
    print("Error reading Excel:", e)

File: inspect_excel.js
const XLSX = require('xlsx');
const path = require('path');

const excelPath = path.join(__dirname, 'Acompanhamento de baixa - MODENS IHS.xlsx');

try {
  console.log('Reading Excel file...');
  const workbook = XLSX.readFile(excelPath);
  console.log('Sheets in Excel:', workbook.SheetNames);

  workbook.SheetNames.forEach(sheetName => {
    const sheet = workbook.Sheets[sheetName];
    // Convert to JSON
    const data = XLSX.utils.sheet_to_json(sheet, { header: 1 });
    console.log(`\n--- Sheet: ${sheetName} ---`);
    console.log(`Total Rows: ${data.length}`);
    if (data.length > 0) {
      console.log('Header Columns:', data[0]);
      console.log('Sample Rows (first 3):');
      for (let i = 1; i <= Math.min(3, data.length - 1); i++) {
        console.log(`Row ${i}:`, data[i]);
      }
    }
  });
} catch (error) {
  console.error('Error reading Excel file:', error);
}
